### DISCUSSION TITLE
CSIRO grassland fires vegetation model

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#12966] [Q&A]
Hi,

I am doing a project on grassland fires and tried to run Case_F19.fds with the vegetation model that was given in the same folder. 

&REAC FUEL='FUEL VAPOR', C=2.10, H=6.20, O=2.16, SOOT_YIELD=0.01, HEAT_OF_COMBUSTION=17425., IDEAL=T /

The line above is the first line in the vegetation model file. When I run the file, I get a warning that says that FUEL VAPOR is not in the table of pre-defined species and therefore the properties of nitrogen are taken. This is obviously not desired. 
The validation guide states that for the CSIRO grassland fires, cellulose is the main component of grass and that cellulose is assumed as the chemical composition of grass. 
I have two questions:
1) if cellulose is taken as the chemical composition of grass (fuel), why is the composition different in the vegetation model?
In the vegetation model  C=2.10, H=6.20, O=2.16 is specified while cellulose is C6H10O5.  
2) Should I name the fuel cellulose instead of FUEL VAPOR? so, FUEL='CELLULOSE', C=6., H=10., O=5., ?

**Comment by bjmvandongen** [comment]
I just tried it with FUEL='CELLULOSE' and that also gives the same warning

**Comment by rmcdermo** [comment]
Please read the warning carefully.  It says that any "undefined properties" are given that of nitrogen.  So, for example, you did not define the viscosity and its functional dependence with temperature.  Unless we have a given species in our database, we need to assign something for these undefined properties, and since N2 is usually the most abundant species, this is a reasonable fallback for these gas phase properties.  Look carefully at the .out file and you will see what properties were assigned to all of the tracked and primitive species in the calculation.
